src/components/Authentication/Login.js

```diff
@@ -50,7 +49,8 @@ const useStyles = makeStyles(theme => ({
   root: {
     borderRadius: 12,
     minWidth: 256,
-    textAlign: "center"
+    textAlign: "center",
```

Comment:
are we going to try to use single quotation marks for all our styling stuff

Reply:
will be fixed when eslint is working